server/pulp/server/db/migrations/0023_importer_tls_storage.py

```diff
@@ -0,0 +1,79 @@
+"""
+This migration inspects all the Importers and writes any TLS certificates or keys they have to local
+storage, as required by the streamer.
+"""
+import errno
+import os
+
+from pulp.server.db import connection
+
+
+LOCAL_STORAGE = os.path.join('/', 'var', 'lib', 'pulp')
+
+
+def migrate():
+    """
+    Write the TLS certificates and keys for each Importer to local storage.
+    """
+    for importer in connection._DATABASE.repo_importers.find():
+        pki_path = _pki_path(importer['repo_id'], importer['importer_type_id'])
+        pem_key_paths = (
+            ('ssl_ca_cert', 'ca.crt'),
+            ('ssl_client_cert', 'client.crt'),
+            ('ssl_client_key', 'client.key'))
+        for key, filename in pem_key_paths:
+            _write_pem_file(pki_path, importer['config'], key, filename)
+
+
+def _mkdir(path):
```

Comment:
suggest rename to `_makedirs` to be explicit about use of `os.makedirs` and `os.mkdir`.